sdcm/sct_events/operator.py

```diff
@@ -58,11 +60,21 @@ def add_info(self: T_log_event, node, line: str, line_number: int) -> T_log_even
                 microsecond=int(milliseconds), tzinfo=datetime.timezone.utc).timestamp()
         except Exception:  # pylint: disable=broad-except
             self.timestamp = time.time()
+
+        self.node = str(node)
+        self.line = line
+        self.line_number = line_number
```

Comment:
IIUC, Dmitry set `self.line_number` to `None` to skip it in `msgfmt`. So, I think, we should keep it `None` here too.

Reply:
removed